src/trex/cli/__init__.py

```diff
@@ -122,6 +122,21 @@ def add_common_arguments(parser, smartseq: bool):
 
     filter_group = parser.add_argument_group("Filter settings")
 
+    filter_group.add_argument(
+        "--min-bases-detected",
+        help="Minimum number of bases detected a cloneID must have. Choose negative values to completely skip removing odd barcodes."
+        "Default: %(default)s",
+        type=int,
+        metavar="INT",
+        default=-1,
+    )
+    filter_group.add_argument(
```

Comment:
I realized now that you add the argument in the `add_common_arguments` function. This means that also the `smartseq2` and `smartseq3` subcommands will have that option, but the code to handle that argument was only added to `run10x`. So the option will be ignored for the other two subcommands.

Reply:
This might be a question for @Leonievb. If these addition is useful for smartseq protocols, then we should include them at some point.